src/autoload.php

```diff
@@ -1,7 +1,7 @@
 <?php
 // This file can be used if you do not use composer to get all the dependencies. 
 //Then you need to download all the dependencies manually and change the first require line accordingly.
-
+include_once '.;C:\php\pear';
```

Comment:
why is this required? also please not that if the file is really mandatory, include_once will only trigger a warning and not an error and the code will break later the in the execution